portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/cpdocumentsandmedia/CPDocumentsandmedia.testcase

```diff
@@ -55,47 +55,10 @@
 		</execute>
 	</command>
 
-	<command name="AddDocument" priority="4">
```

Comment:
Oh maaan, I did it: https://github.com/liferay/liferay-portal/commit/ca4874d81e75f58eb1bb94c3b301199ad87d2b8f